### PR TITLE
Fix and Publish application on Rinkeby network 

### PR DESCRIPTION
**Pull request by blackinno**
- Fix Aragon application fetch error data
- Update Abi for encodeEventSignature at `process/deployer/encodeEventSignature.js`.
- Test and Publish on the Rinkeby 

 Closed https://github.com/mesg-foundation/aragon/issues/9
 Closed https://github.com/mesg-foundation/aragon/issues/11

### CODE REVIEW

**Comment by blackinno** [comment]
@NicolasMahe Could you redeploy process deployer? The application is fixed and run correctly. You can on this [link](https://rinkeby.aragon.org/#/0x4f0B96b4F0C8d88432d4B66F3413bC8Fac722207/0xd2474fbfb356c2fd1d9b5dc2439f97d2e5c3d351/)

**Comment by blackinno** [comment]
Fix issue https://github.com/mesg-foundation/aragon/issues/13. Done.

Preview link: https://rinkeby.aragon.org/#/0x702E819000FC3Be0dadC99480401bD94DBD5b322/0xf390905ab56bf9ed27fbc9d24ead1b46c8053ec5/

File: contracts/ProcessApp.sol [hunk 1]
```diff
@@ -39,36 +41,40 @@ contract ProcessApp is AragonApp {
...
+            serviceName,
+            data
         );
     }
 
     /**
      * @notice Desactivate a process
      */
-    function desactivate(uint256 index) external auth(DESACTIVATE_ROLE) {
+    function deacivate(uint256 index, address projectName) external auth(DEACTIVATE_ROLE) {
```
Comment (NicolasMahe) [inline contracts/ProcessApp.sol:74]:
```suggestion
    function deactivate(uint256 index, address projectName) external auth(DEACTIVATE_ROLE) {
```

File: app/src/components/DataConnection/TableConnection.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                <b>DEACTIVATED</b>
+              </span>
+            )}
+          </div>
+        ]
+      }}
+      renderEntryActions={({ entity, index }) => {
+        return (
+          <ContextMenu zIndex={1}>
+            <ContextMenuItem onClick={() => api.deacivate(index, organization.appAddress.toLowerCase()).toPromise()}>
```
Comment (NicolasMahe) [inline app/src/components/DataConnection/TableConnection.js:130]:
```suggestion
            <ContextMenuItem onClick={() => api.deactivate(index, organization.appAddress.toLowerCase()).toPromise()}>
```

**Pull request merged**